src/java.base/share/classes/jdk/internal/util/random/RandomSupport.java

```diff
@@ -1425,75 +1419,18 @@ public static double computeNextGaussian(RandomGenerator rng) {
          * satisfy the interface RandomGenerator.  An implementation of this
          * interface need only extend this class and provide implementations
          * of six methods: nextInt, nextLong, and nextDouble (the versions
-         * that take no arguments) and makeIntsSpliterator,
-         * makeLongsSpliterator, and makeDoublesSpliterator.
+         * that take no arguments).
          *
          * File organization: First the non-public abstract methods needed
          * to create spliterators, then the main public methods.
          */
 
         /**
-         * Explicit constructor.
+         * No instances.
          */
-        protected AbstractSpliteratorGenerator() {
+        private AbstractSpliteratorGenerator() {
         }
 
```

Comment:
This comment is wrong now since there are instances of this class (really, subclasses). The constructor should probably simply be removed.